tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html

```diff
@@ -0,0 +1,229 @@
+            <p>Probable causes:</p>
+            <ul>
+              <li>
+                You haven’t written any precision–recall data to your event
+                files.
+              </li>
+              <li>
+                TensorBoard can’t find your event files.
+              </li>
+            </ul>
+            <p>
+            If you’re new to using TensorBoard, and want to find out how
+            to add data and set up your event files, check out the
+            <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md">README</a>
+            and perhaps the <a href="https://www.tensorflow.org/get_started/summaries_and_tensorboard">TensorBoard tutorial</a>.
+            <p>
+            If you think TensorBoard is configured properly, please see
+            <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md#my-tensorboard-isnt-showing-any-data-whats-wrong">the section of the README devoted to missing data problems</a>
+            and consider filing an issue on GitHub.
+          </div>
+        </template>
+        <template is="dom-if" if="[[!_dataNotFound]]">
+          <paper-input
+            no-label-float
+            label="Filter tags (regular expressions supported)"
+            value="{{_tagFilter}}"
+            class="search-input"
+          >
+            <iron-icon prefix icon="search"></iron-icon>
+          </paper-input>
+          <template is="dom-repeat" items="[[_categories]]" as="category">
+            <tf-category-pane category="[[category]]">
+              <tf-paginated-view
+                items="[[category.items]]"
+                pages="{{category._pages}}"
+              >
+                <template is="dom-repeat" items="[[category._pages]]" as="page">
+                  <template is="dom-if" if="[[page.active]]">
+                    <div class="layout horizontal wrap">
+                      <template is="dom-repeat" items="[[page.items]]">
+                        <tf-pr-curve-card
+                          runs="[[item.runs]]"
+                          tag="[[item.tag]]"
+                          request-manager="[[_requestManager]]"
+                          run-to-step-cap="[[_runToStep]]"
+                        ></tf-pr-curve-card>
+                      </template>
+                    </div>
+                  </template>
+                </template>
+              </tf-paginated-view>
+            </tf-category-pane>
+          </template>
+        </template>
+      </div>
+    </tf-dashboard-layout>
+
+    <style include="dashboard-style"></style>
+    <style>
+      #instructions {
+        width: 300px;
+      }
+
+      .tooltip-block-container {
+        display: inline-block;
+      }
+
+      .no-data-warning {
+        max-width: 540px;
+        margin: 80px auto 0 auto;
+      }
+
+      .run-step-combo .run-text {
+        margin: 10px 0 0 0;
+      }
+    </style>
+  </template>
+
+  <script>
+  "use strict";
```

Comment:
[@jart says](https://github.com/tensorflow/tensorboard/pull/112#discussion_r123370125) that you shouldn't `use strict`.